doc/iterations/2025.4/repeated_tool_invocation.adoc

```diff
@@ -0,0 +1,79 @@
+= (S) Make it easier to invoke the same tool multiple times in a row
```

Comment:
Small nitpick but on macos both alt and cmd exists (alt is just named either alt or option) so I think mentioning just alt is enough since I don't think there is a reason to have alt+click on linux/windows and something other than alt+click on mac.